carina-webdriver/src/main/java/com/qaprosoft/carina/core/foundation/webdriver/decorator/ExtendedWebElement.java

```diff
@@ -1537,18 +1537,31 @@ public void doType(String text) {
 				DriverListener.setMessages(Messager.KEYS_CLEARED_IN_ELEMENT.getMessage(getName()),
 						Messager.KEYS_NOT_CLEARED_IN_ELEMENT.getMessage(getNameWithLocator()));
 				element.clear();
+				
+				
+				if (decryptedText.equals(text)) {
```

Comment:
let's introduce new textLog as string which is assigned inside "if" operator. But keep single DriverListener.sendMessage call